### PR TITLE
added ML algorithms | ReadMe.md | Contribution.md

### PR DESCRIPTION
**Pull request by hritik5102**
You can see I added various ML algorithms in the PR.

* [x] Python Basics
* [x] numpy
* [x] Matplotlib
* [x] Pandas
* [x] Scikit learn
* [x] Linear regression
* [x] Cosine similarity
* [x] Naive bayes
* [x] Neural network MNIST dataset
* [x] Natural language processing
* [x] ReadMe.md
* [x] Contributing.md

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Jupyter ![](https://img.shields.io/github/forks/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/stars/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/watchers/TheAlgorithms/Jupyter?style=social) <br>
```
Comment (cclauss) [inline ReadME.md]: Capitolization: `README.md` for consistency with other repos.

Comment (hritik5102) [reply ReadME.md]: Ok

**Comment by hritik5102** [comment]
@cclauss kindly review the PR, it has been modified.

File: ReadME.md (hunk 1 above)
Comment (hritik5102) [reply ReadME.md]: @cclauss kindly review the PR, it has been modified.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## Jupyter ![](https://img.shields.io/github/forks/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/stars/TheAlgorithms/Jupyter?style=social) ![](https://img.shields.io/github/watchers/TheAlgorithms/Jupyter?style=social) <br>
+
+![](https://img.shields.io/github/repo-size/TheAlgorithms/Jupyter) ![](https://img.shields.io/github/downloads/TheAlgorithms/Jupyter/total)<br>
+![](https://img.shields.io/github/issues/TheAlgorithms/Jupyter?color=green) ![](https://img.shields.io/github/issues-pr/TheAlgorithms/Jupyter?color=green) ![](https://img.shields.io/github/last-commit/TheAlgorithms/Jupyter) ![](https://img.shields.io/github/contributors/TheAlgorithms/Jupyter)
+
+## Clone git repository
+
+```sh
+    $ Git clone "https://github.com/TheAlgorithms/Jupyter.git"
```
Comment (cclauss) [inline README.md]: Git --> git

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+You can run and edit the algorithms or contribute to them using [Gitpod.io](https://www.gitpod.io/), a free online development environment, with a single click.
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](http://gitpod.io/#https://github.com/TheAlgorithms/Jupyter)
+
+## Contributing
+
+Please read [Contributing.md](./CONTRIBUTION.md/Contributing.md) for information on how to contribute to Jupyter.
```
Comment (cclauss) [inline README.md]: Let's remove the `CONTRIBUTION.md` directory and put `Contributing.md` at the root.

**Comment by hritik5102** [comment]
> A few minor change requests.

File: README.md (hunk 2 above)
Comment (hritik5102) [reply README.md]: Solved sir 👍

File: README.md (hunk 3 above)
Comment (hritik5102) [reply README.md]: Modified sir 👍

**Pull request merged**